tests/unit/python-utilities-valid.test.ts

```diff
@@ -80,4 +44,40 @@ describe("Testing pythonUtilities (valid)", () => {
     },
     timeout
   );
+
+  test("should create and send a diagnostic objects for tst", () => {
+    const diagnostic: Diagnostic = {
```

Comment:
Have a look if you can make a utility function for diagnostics stuff - quite some things seem to be repeated from `ct-completion.test.ts`

Reply:
Added setupDiagnosticTest in 314cad1a860f1d3471af84c9d50e0bfc08afd811 .